### PR TITLE
LOC CHECKIN | dotnet/project-system dev16.2.x  | 20190621

### PR DESCRIPTION
**Pull request by v-zbsail**

### CODE REVIEW

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/xlf/VSPackage.zh-Hant.xlf [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
-  <file datatype="xml" source-language="en" target-language="zh-Hant" original="../VSPackage.resx">
+  <file datatype="xml" source-language="en" target-language="zh-HANT" original="../VSPackage.resx">
```
Comment (drewnoakes) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/xlf/VSPackage.zh-Hant.xlf]: Is this change ok?

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Items/xlf/EditorConfigFiles.BrowseObject.xaml.de.xlf [hunk 2]
```diff
@@ -4,97 +4,97 @@
...
       <trans-unit id="Rule|EditorConfigFiles|Description">
         <source>File Properties</source>
-        <target state="new">File Properties</target>
+        <target state="translated">Dateieigenschaften</target>
         <note />
       </trans-unit>
       <trans-unit id="Rule|EditorConfigFiles|DisplayName">
         <source>.editorconfig File</source>
-        <target state="new">.editorconfig File</target>
+        <target state="translated">EDITORCONFIG-Datei</target>
```
Comment (drewnoakes) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Items/xlf/EditorConfigFiles.BrowseObject.xaml.de.xlf]: I don't know German but I would think `.editorconfig` should be kept verbatim here. It is the culture-invariant name of a file. Other translations here keep it.

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Items/xlf/EditorConfigFiles.BrowseObject.xaml.ru.xlf [hunk 3]
```diff
@@ -4,97 +4,97 @@
...
       <trans-unit id="Rule|EditorConfigFiles|Description">
         <source>File Properties</source>
-        <target state="new">File Properties</target>
+        <target state="translated">Свойства файла</target>
         <note />
       </trans-unit>
       <trans-unit id="Rule|EditorConfigFiles|DisplayName">
         <source>.editorconfig File</source>
-        <target state="new">.editorconfig File</target>
+        <target state="translated">Файл EDITORCONFIG</target>
```
Comment (drewnoakes) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Items/xlf/EditorConfigFiles.BrowseObject.xaml.ru.xlf]: Another capitalisation of `.editorconfig`.

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/xlf/VSPackage.zh-Hant.xlf (hunk 1 above)
Comment (josefhosek) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/xlf/VSPackage.zh-Hant.xlf]: This wasn't expected, we'll fix it.

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Items/xlf/EditorConfigFiles.BrowseObject.xaml.de.xlf (hunk 2 above)
Comment (josefhosek) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Items/xlf/EditorConfigFiles.BrowseObject.xaml.de.xlf]: This is the correct way how German and Russian translate file extension (.abc file -> ABC-Datei for German for example). So this is expected.

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Items/xlf/EditorConfigFiles.BrowseObject.xaml.ru.xlf (hunk 3 above)
Comment (josefhosek) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Items/xlf/EditorConfigFiles.BrowseObject.xaml.ru.xlf]: Answered in above comment.

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/xlf/VSPackage.zh-Hant.xlf (hunk 1 above)
Comment (drewnoakes) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/xlf/VSPackage.zh-Hant.xlf]: FYI there are other target language changes in the PR too.

**Comment by davkean** [comment]
@tmeschter Can you take a look over this and see if these changes are expected?

File: src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/xlf/VSPackage.zh-Hant.xlf (hunk 1 above)
Comment (tmeschter) [reply src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/xlf/VSPackage.zh-Hant.xlf]: The casing shouldn't matter on our end.

**Comment by tmeschter** [comment]
@davkean If you're referring to the `target-language` changes they are innocuous. XliffTasks tends to create .xlf files where the `target-language` value has mixed casing, like `zh-Hans`. I suspect some of the loc team's tooling has previously substituted in values like `zh-HANS`, so the changes in this PR undo most of that and return to the original values. Again, it shouldn't matter for our purposes.

**Pull request merged**